builders/nanoserver-1809/README.md

```diff
@@ -0,0 +1,21 @@
+# Sample Nano Server 1809 Builder
+
+### Prerequisites
+* [Pack](https://buildpacks.io/docs/install-pack/)
+
```

Comment:
Maybe a `* Windows 10 Enterprise 2019 or Windows Server 1809`? That might also be implied by the title of the README